skimage/feature/_canny.py

```diff
@@ -164,7 +166,7 @@ def canny(image, sigma=1., low_threshold=None, high_threshold=None, mask=None,
 
     if mask is None:
         mask = np.ones(image.shape, dtype=bool)
-    fsmooth = lambda x: gaussian_filter(x, sigma, mode='constant')
+    fsmooth = lambda x: gaussian(x, sigma, mode='constant', multichannel=False)
     smoothed = smooth_with_function_and_mask(image, fsmooth, mask)
```

Comment:
This change is fine to me. Docstring declares no support for multichannel images and function has no `multichannel` parameter yet.